internal/venice-common/src/main/java/com/linkedin/venice/meta/Store.java

```diff
@@ -42,7 +42,7 @@ public interface Store {
 
   long DEFAULT_RT_RETENTION_TIME = TimeUnit.DAYS.toMillis(5);
 
-  int DEFAULT_BATCH_GET_LIMIT = 150;
+  int DEFAULT_BATCH_GET_LIMIT = 1000;
```

Comment:
1. Doesn't TC default to 500?
2. Lets add an unit test for the class accessing this so this doesn't get unintenionally modified?